### PR TITLE
[Sprint 1]Feature Create Header In Main

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**
Feature Create Header In Main

### CODE REVIEW

File: app/src/main/res/drawable/border_button.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    <stroke
+        android:color="@android:color/black"
+        android:width="1dp"/>
+    <corners android:radius="40dp"/>
+    <solid android:color="@android:color/white"/>
+    <padding
+        android:bottom="10dp"
+        android:left="10dp"
+        android:right="10dp"
+        android:top="10dp"/>
```
Comment (Tuanlvt) [inline app/src/main/res/drawable/border_button.xml]: hard code

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -11,13 +14,90 @@
...
+                    android:background="@color/colorPrimary"
+                    >
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:gravity="center_vertical"
+                        android:orientation="horizontal"
+                        >
+                        <FrameLayout
+                            android:id="@+id/search"
```
Comment (Tuanlvt) [inline app/src/main/res/layout/activity_main.xml]: remove it

File: app/src/main/res/drawable/border_button.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <stroke
+        android:color="@android:color/black"
+        android:width="1dp"/>
```
Comment (daolq3012) [inline app/src/main/res/drawable/border_button.xml]: hard code

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -11,13 +14,89 @@
...
 
-<android.support.constraint.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context="com.framgia.fbook.screen.main.MainActivity"
-    >
-</android.support.constraint.ConstraintLayout>
+    <android.support.design.widget.CoordinatorLayout
```
Comment (daolq3012) [inline app/src/main/res/layout/activity_main.xml]: pls optimize this layout, there is too much layout container!
Pls use 1 container layout

**Pull request merged**